src/Servers/HttpSys/samples/TlsFeaturesObserve/Program.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Server.HttpSys;
+using Microsoft.Extensions.Hosting;
+using TlsFeatureObserve;
+using TlsFeaturesObserve.HttpSys;
+
+HttpSysConfigurator.ConfigureCacheTlsClientHello();
```

Comment:
What is this?

Reply:
Fetching TLS client hello feature from HTTP.SYS works only if a pre-configuration was done for ssl cert binding and netsh ssl config beforehand. User would need to set service configuration via [HttpSetServiceConfiguration](https://learn.microsoft.com/en-gb/windows/win32/api/http/nf-http-httpsetserviceconfiguration) with `HTTP_SERVICE_CONFIG_SSL_FLAG_ENABLE_CACHE_CLIENT_HELLO` flag.

That is and should not be a part of ASP.NET app, but I added that for "sample app" for simplicity of reproduction.